include/nbl/video/SPhysicalDeviceFeatures.h

```diff
@@ -0,0 +1,563 @@
+    //VkBool32           subgroupBroadcastDynamicId;    // if Vulkan 1.2 is supported
+
+
+
+
+    /* Vulkan 1.3 Core */
+
+    //VkBool32           robustImageAccess;                 //  or VK_EXT_image_robustness
+    
+    //  or VK_EXT_inline_uniform_block:
+    //VkBool32           inlineUniformBlock;
+    //VkBool32           descriptorBindingInlineUniformBlockUpdateAfterBind;
+    
+    //VkBool32           pipelineCreationCacheControl;      // or VK_EXT_pipeline_creation_cache_control
+    //VkBool32           privateData;                       // or VK_EXT_private_data
+    //VkBool32           shaderDemoteToHelperInvocation;    // or VK_EXT_shader_demote_to_helper_invocation
+    //VkBool32           shaderTerminateInvocation;         // or VK_KHR_shader_terminate_invocation
+    
+    // or VK_EXT_subgroup_size_control
+    bool subgroupSizeControl  = false;
+    bool computeFullSubgroups = false;
+    
+    //VkBool32           synchronization2;                      // or VK_KHR_synchronization2
+    //VkBool32           textureCompressionASTC_HDR;            // or VK_EXT_texture_compression_astc_hdr
+    //VkBool32           shaderZeroInitializeWorkgroupMemory;   // or VK_KHR_zero_initialize_workgroup_memory
+    //VkBool32           dynamicRendering;                      // or VK_KHR_dynamic_rendering
+    //VkBool32           shaderIntegerDotProduct;               // or VK_KHR_shader_integer_dot_product
+    //VkBool32           maintenance4;                          // ! Doesn't make sense to expose, too vulkan specific // or VK_KHR_maintenance4
+
+
+
+
+    /* Vulkan Extensions */
+
+    /* CoherentMemoryFeaturesAMD *//* VK_AMD_device_coherent_memory */
+    //VkBool32           deviceCoherentMemory;
+
+    /* VK_AMD_shader_early_and_late_fragment_tests *//* couldn't find the struct/extension in latest vulkan headers */
+
+    /* RasterizationOrderAttachmentAccessFeaturesARM *//* VK_ARM_rasterization_order_attachment_access */
+    //VkBool32           rasterizationOrderColorAttachmentAccess;
+    //VkBool32           rasterizationOrderDepthAttachmentAccess;
+    //VkBool32           rasterizationOrderStencilAttachmentAccess;
+    
+    /* 4444FormatsFeaturesEXT *//* VK_EXT_4444_formats */
+    //VkBool32           formatA4R4G4B4;
+    //VkBool32           formatA4B4G4R4;
+    
+    /* ASTCDecodeFeaturesEXT *//* VK_EXT_astc_decode_mode */
+    //VkFormat           decodeMode;
+
+    /* BlendOperationAdvancedFeaturesEXT *//* VK_EXT_blend_operation_advanced */
+    //VkBool32           advancedBlendCoherentOperations;
+
+    /* BorderColorSwizzleFeaturesEXT *//* VK_EXT_border_color_swizzle */
+    //VkBool32           borderColorSwizzle;
+    //VkBool32           borderColorSwizzleFromImage;
+
+    /* VK_EXT_buffer_device_address *//* HAS KHR VERSION */
+
+    /* ColorWriteEnableFeaturesEXT *//* VK_EXT_color_write_enable */
+    //VkBool32           colorWriteEnable;
```

Comment:
trivial to support in GL , but would need new commandbuffer methods, etc.  so leave as a TODO